connbeat.yml

```diff
@@ -0,0 +1,24 @@
+connbeat:
+  expose_process_info: true
+  expose_cmdline: true
+  expose_environ: true
+  aggregation: 10s
+  poll_interval: 2s
+  enable_tcp_diag: true
+
+output:
+  http:
+    hosts: ["${STSURL}"]
+
+  #kafka:
+  #  hosts: ["localhost:9092"]
+  #  topic: connbeat
+
+  #elasticsearch:
+  #  hosts: ["localhost:9200"]
+    #template:
+    #  name: "connbeat"
+    #  path: "connbeat.template.json"
+
+  console:
+    pretty: true
```

Comment:
I guess we don't need to log to the console when we deploy via the sts-agent package?